deps/install.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
```

Comment:
Can we document how .conf files should be structured somewhere (maybe `deps/README.md`)? If you know about bash, its possible to decipher exactly how this script uses the `.conf` files, but if you don't it can be quite hard I think

Reply:
Good point. Let me put together some documentation.

Reply:
Added deps/README.md